src/tests/dbus-tests/test_10_basic.py

```diff
@@ -5,9 +5,24 @@
 class StoragedBaseTest(storagedtestcase.StoragedTestCase):
     '''This is a base test suite'''
 
+    fedora_modules = ['Bcache', 'BTRFS', 'ISCSI.Initiator', 'LVM2', 'ZRAM']
+    rhel_modules = ['BTRFS', 'ISCSI.Initiator', 'LVM2', 'ZRAM']
+
     def setUp(self):
         self.manager_obj = self.get_object('/Manager')
 
+    def _get_modules(self):
+        content = self.read_file('/etc/os-release')
+        release = {key: value for (key, value) in [line.split('=') for line in content.split('\n') if line]}
+        distro = release['ID'].replace('"', '')
+
+        if distro == 'fedora':
+            return self.fedora_modules
+        elif distro in ('redhat', 'centos'):
+            return self.rhel_modules
+        else:
+            self.fail('Unknown distribution %s' % distro)
```

Comment:
I think this should check for rhel/centos we know to have some limitations and fall back to the full list of modules otherwise. It's better to just add exceptions than keep adding more and more things.